modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field_feedback_support.js

```diff
@@ -3,7 +3,7 @@ AUI.add(
 	function(A) {
 		var Lang = A.Lang;
 
-		var TPL_ERROR_MESSAGE = '<div class="help-block">{errorMessage}</div>';
+		var TPL_ERROR_MESSAGE = '<div class="error-message help-block">{errorMessage}</div>';
```

Comment:
Changes like this often will affect our test locators. See https://github.com/brianchandotcom/liferay-portal/pull/40701/commits/55f1202dd7d4b25cd5c22461080c669a1dc4b57a